browse/services/object_store/fileobj.py

```diff
@@ -205,10 +227,11 @@ def size(self) -> int:
             # file.  That won't record file sizes larger than 4Gb and there may
             # be other quirks.  So for now we get it by reading and unzipping
             # the whole file.
-            with self.open(mode="rb") as unzipped_fh:
-                size = unzipped_fh.seek(0, io.SEEK_END)
-                self._size = size
-                return self._size
+            unzipped_fh = self.open(mode="rb")
+            size = unzipped_fh.seek(0, io.SEEK_END)
+            unzipped_fh.close()
+            self._size = size
+            return self._size
```

Comment:
Why did this change?  The "with ..." wraps the indented block with a try / finally close, and the new code is not similarly protected against thrown errors.